dms/__manifest__.py

```diff
@@ -4,17 +4,18 @@
 {
     "name": "Document Management System",
     "summary": """Document Management System for Odoo""",
-    "version": "15.0.1.8.2",
+    "version": "16.0.1.0.0",
     "category": "Document Management",
     "license": "LGPL-3",
     "website": "https://github.com/OCA/dms",
     "author": "MuK IT, Tecnativa, Odoo Community Association (OCA)",
     "depends": [
-        "web_drop_target",
+        # "web_drop_target", #TODO only exist in v15
         "mail",
         "http_routing",
         "portal",
-        "mail_preview_base",
+        # "mail_preview_base", #TODO only exist in v15
+        "base",
```

Comment:
If `web_drop_target` and `mail_preview_base` are not needed dependencies can be removed (do not mark them as TODO. 

After testing it functionally in runboat, it seems that they are no longer needed and that the necessary code has been added to `dms`, good job!!!!

**Note**: How would the compatibility be if for example you want to preview an audio with `mail_preview_audio`?

Reply:
Unfortunately, there has never been support for Office/LibreOffice formats. The module `mail_preview_base` served for clicking on the icon of the kanban view of the attachments/dms files and being able to preview them, but not expanding the supported files (PDF, images, videos and text). `mail_preview_base_audio` expanded the support for the audio files, although I think now there's support for them in core as well.

Reply:
If we want to preview more data types, I suggest using the module `attachment_preview` instead. If anyone wants this module in v16, please open an issue. I could migrate it if I have time for that. @pedrobaeza, do you know if anyone can review this PR [attachment_preview PR 361](https://github.com/OCA/knowledge/pull/361) and merge it if everything is ok?

Reply:
Well, adding a glue module between 2 can be a good option for viewing them, but I would like to go further with a direct edition feature using some of the available server options (LibreOffice online or OnlyOffice integration with Odoo - https://www.onlyoffice.com/es/office-for-odoo.aspx -).

Reply:
Oh that's right I confused that with the `attachment_preview` module. 
It'd be a nice thing to have for sure!

OnlyOffice could be nice I guess, but it seems it would be some order(s) of magnitude larger in effort. And it doesn't invalidate that a direct preview embedded would probably still be useful, unless OnlyOffice could instantly load in an iframe without much disruption of the UX, which doesn't seem likely.